docs/tex/HACCP.tex

```diff
@@ -5,28 +5,55 @@ \section{Hazard Analysis and Critical Control Point (HACCP) Plan}
 \subsection{Food Production System Description}
 % Incl. flowchart
 
+
 \subsection{Critical Points}
+%assembly: only approved parts that have been checked for leaks, contamination, materials, air-tightness
+
+%seeds: vetted and tested breed from an approved supplier, not opened until moment of planting, food safety steps followed while doing so (hand washing, gloves, masks, hairnet...)
+
+%growth medium: tested for bacteria/pests, handled carefully before installation
+
+%system inputs: filtered air, clean water, properly sourced nutrients, all tested by appropriate standards
+
+%maintenance: plants properly isolated when non-food safe materials are present---i.e. if LED boards need to be swapped, if something needs to be greased, if something needs to be glued
+
+%harvesting: all food safety guidelines to be followed - hand washing, gloves, masks, washing, etc.
+
+%re-planting: checking for no old growth medium, sanitizing growth tray with food-safe sanitizer
 
 \subsubsection{Critical Point A}
-% Hazard description
-% Critical limits
-% Monitoring procedures
-% Deviation procedures
-% Associated documents
+\textbf{Hazard Description}
```

Comment:
What is the variable being controlled? What is the function of the control point?